kafka-rest/src/main/java/io/confluent/kafkarest/ratelimit/RateLimitModule.java

```diff
@@ -37,6 +37,12 @@ protected void configure() {
         .to(RequestRateLimiter.class)
         .in(Singleton.class);
 
+    bindFactory(RequestRateLimiterGenericFactory.class)
```

Comment:
I think you'll need a new factory RequestRateLimiterPerLKCFactory here, like we do for the producer global vs per lkc, as you need to be able to set different limits on each

Eg global could be 25, but per lkc could be 10.

Reply:
That makes sense, I'll add a new factory